### DISCUSSION TITLE
Return auth user on signup

### DISCUSSION THREAD

**Discussion by AshtonMartinez** [supabase/supabase#17681] [Questions]
I am wanting to get a reference to a newly created auth user so I can set a relation in a public users table, but I can not figure out how to get a new created auth user. Can you only get a reference to the signed in auth user for that session?

**Comment by GaryAustin1** [comment]
Maybe you can explain a bit more about what your are trying to do.  A createClient() "object" only works with one user signed in at a time.  How did you create the user?    How are you trying to use this new user while you have a signed in user?  Your RLS determines if only a signed in user can access the public users table or any authenticated user or any anon user.
